templates/corporate/upgrade.html

```diff
@@ -146,7 +146,9 @@ <h1>$0</h1>
                             {% endif %}
                             {% if page_params.flat_discounted_months > 0 %}
                             <br/>
-                            Discount: $<span class="flat-discounted-price"></span>/month off <i class="billing-page-discount">({{ page_params.flat_discounted_months }} {{ "month" if discounted_months_left == 1 else "months" }} remaining)</i>
+                            <span class="flat-discount-minus-sign">−</span>
+                            $<span class="flat-discounted-price"></span>/month off <i class="billing-page-discount">({{ page_params.flat_discounted_months }} {{ "month" if discounted_months_left == 1 else "months" }} remaining)</i>
+                            <div class="flat-discount-separator">-------------------</div>
```

Comment:
Is `discounted_months_left` set in this template's context, or should this be referencing `page_params.flat_discounted_months` instead?